scalar/README.md

```
@@ -0,0 +1,129 @@
+---
+id: scalar
+title: Scalar
+---
+
+# Scalar
+
+Scalar middleware for [Fiber](https://github.com/gofiber/fiber). The middleware handles Scalar UI.
+
+**Note: Requires Go 1.23.0 and above**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)